
<launch>
    <remap from="cmd_vel" to="/nav/cmd_vel_nav"/>
    <!-- <remap from="/map" to="/rtabmap/proj_map"/> -->
    <node pkg="move_base" type="move_base" respawn="false" name="move_base" output="screen">
        <!-- <param name="controller_frequency" value="1.42857143"/> -->
        <param name="controller_frequency" value="5"/>
        <param name="base_local_planner" value="eband_local_planner/EBandPlannerROS"/>
        <!-- <param name="base_local_planner" value="dwa_local_planner/DWAPlannerROS"/> -->
        <rosparam file="$(find walker_nav)/configs/walker_costmap_common.yaml" command="load" ns="global_costmap" />
        <rosparam file="$(find walker_nav)/configs/walker_costmap_common.yaml" command="load" ns="local_costmap" />
        <rosparam file="$(find walker_nav)/configs/walker_costmap_local.yaml" command="load" />
        <rosparam file="$(find walker_nav)/configs/walker_costmap_global.yaml" command="load" />
        <rosparam file="$(find walker_nav)/configs/walker_planer_local.yaml" command="load" />
    </node>
 </launch> 